helm/net-exporter-chart/templates/daemonset.yaml

```diff
@@ -44,6 +44,9 @@ spec:
         args:
         - "-namespace={{ .Values.namespace }}"
         - "-timeout={{ .Values.timeout }}"
+        {{ if (.Values.Installation) }}
+        - "-disable-dns-tcp-check={{ .Values.Installation.V1.Monitoring.NetExporter.DisableDNSTCPCheck }}"
```

Comment:
Isn't it better to have something more structured? Like `NetExporterd.DNSCheck.TCP.Enabled`?

Reply:
why not. Is possible possible to have other value there?

Reply:
Do we already have that defined in installations?

Reply:
I have a PR in place

Reply:
but nothing for net-exporter yet.

Reply:
> why not. Is possible possible to have other value there?

Yes we can have `NetExporter.DNSCheck.UDP.Enabled` for example.